src/CONST.js

```diff
@@ -67,8 +67,8 @@ const CONST = {
     },
     MESSAGES: {
```

Comment:
It's very odd to me that, out of all the strings in the app, just these two messages and the pronouns are included in this CONST file. I think that there are two ways we could handle this:

1. Move _all_ strings into a `UI_STRINGS.js` constant file, that looks sort of like this:

    ```javascript
    CANCEL: 'common.cancel',
    ...
    NO_PHONE_NUMBER: 'noPhoneNumberMessage',
    ...
    
    ```
    
    Where each constant provides a `lodashGet` path to a string in one of our language files.
    
2. Just remove these constants from `CONST.js` and use their `lodashGet` path directly in the component that's rendering them (like you have for most other components in this PR).

@iwiznia, do you have any opinion on which is better? I would lean towards (2) just because it's a bit simpler, but in Mobile-Expensify we had a `UIStrings` constants file.